brands/shop/travel_agency.json

```diff
@@ -83,6 +83,15 @@
       "shop": "travel_agency"
     }
   },
+  "shop/travel_agency|Selectour": {
+    "countryCodes": ["fr"],
+    "tags": {
+      "brand": "Selectour",
+      "brand:wikidata": "Q3478073",
+      "name": "Selectour",
```

Comment:
also add `"brand:wikipedia": "fr:Selectour"`

Reply:
done, it was supposed to be in the patch.
Thanks for spotting that.